[2025-05-08T22:13:59.821+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-08T22:13:59.861+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.Load_reviews1 manual__2025-05-08T22:12:11.230444+00:00 [queued]>
[2025-05-08T22:13:59.878+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.Load_reviews1 manual__2025-05-08T22:12:11.230444+00:00 [queued]>
[2025-05-08T22:13:59.880+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-08T22:13:59.907+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): Load_reviews1> on 2025-05-08 22:12:11.230444+00:00
[2025-05-08T22:13:59.922+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=310) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-08T22:13:59.928+0000] {standard_task_runner.py:72} INFO - Started process 332 to run task
[2025-05-08T22:13:59.927+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'Load_reviews1', 'manual__2025-05-08T22:12:11.230444+00:00', '--job-id', '907', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmpsq88vkv0']
[2025-05-08T22:13:59.930+0000] {standard_task_runner.py:105} INFO - Job 907: Subtask Load_reviews1
[2025-05-08T22:14:00.021+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.Load_reviews1 manual__2025-05-08T22:12:11.230444+00:00 [running]> on host ff2d23694728
[2025-05-08T22:14:00.200+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='elkhalfimaryame@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='Load_reviews1' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T22:12:11.230444+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-08T22:12:11.230444+00:00'
[2025-05-08T22:14:00.206+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-08T22:14:00.207+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-08T22:14:00.208+0000] {logging_mixin.py:190} INFO - Current task name:Load_reviews1 state:running start_date:2025-05-08 22:13:59.862129+00:00
[2025-05-08T22:14:00.210+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-08T22:14:00.211+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-08T22:14:00.254+0000] {logging_mixin.py:190} INFO - Current directory: /opt/***/dags
[2025-05-08T22:14:00.256+0000] {logging_mixin.py:190} INFO - Data path: /opt/***/dags/input/reviews.csv
[2025-05-08T22:14:00.297+0000] {base.py:84} INFO - Retrieving connection 'postgres_defaut'
[2025-05-08T22:14:00.478+0000] {warnings.py:112} WARNING - /opt/***/dags/storeData.py:544: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql_query(query, conn)

[2025-05-08T22:14:00.694+0000] {logging_mixin.py:190} INFO - Data stored in PostgreSQL.
[2025-05-08T22:14:00.698+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-08T22:14:00.716+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-08T22:14:00.717+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=Load_reviews1, run_id=manual__2025-05-08T22:12:11.230444+00:00, execution_date=20250508T221211, start_date=20250508T221359, end_date=20250508T221400
[2025-05-08T22:14:00.758+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-08T22:14:00.761+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-08T22:14:00.763+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-08 22:12:11.268715+00:00
[2025-05-08T22:14:00.764+0000] {logging_mixin.py:190} INFO - Task hostname:ff2d23694728 operator:PythonOperator
[2025-05-08T22:14:00.791+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-08T22:14:00.852+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-08T22:14:00.855+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
